tests/io/pwmat/test_inputs.py

```diff
@@ -129,10 +131,10 @@ def test_write_file(self):
         assert tmp_high_symmetry_points_str == high_symmetry_points.get_str()
 
 
-def test_err_msg_on_seekpath_not_installed(monkeypatch):
```

Comment:
Should I understand correctly, this doesn't really patch the import, but either:
- passes directly if `seekpath` is available, or
- capture error if not